packages/block-editor/src/store/actions.js

```diff
@@ -1591,7 +1591,23 @@ export function updateBlockListSettings( clientId, settings ) {
  * @return {Object} Action object
  */
 export function updateSettings( settings ) {
-	return __experimentalUpdateSettings( settings, {
+	let updatedSettings = settings;
+
+	if ( Object.hasOwn( updatedSettings, '__unstableIsPreviewMode' ) ) {
```

Comment:
I think we should move this into `__experimentalUpdateSettings` because that one can be triggered used from the "settings" prop of the existing editor I believe no?

Reply:
I was between that and doing it like I've done it here. I'll update.